### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Rope-master** [autti/abraham#66]
Do you have a complete dbc file ?

**Comment by ingenieroariel** [comment]
there is one in the comma ai / opendbc repo, it could be missing key
signals but it looks to be more than 90% complete, I tested that one on my
car

**Comment by Rope-master** [comment]
You are right, but the dbc file only contains the signals used by comma ai, and not original the signal of the car.
